docs/guides/mysql/pitr/restic/same-db/archiver.md

```
@@ -0,0 +1,538 @@
+    -e "select count(*) from demo.demo_table;"
+
+$ kubectl exec -it -n demo mysql-0 -- mysql -uroot -p$MYSQL_ROOT_PASSWORD \
+    -e "select member_host, member_state from performance_schema.replication_group_members;"
+```
+
+All three members should report `ONLINE`.
+
+## Cleaning up
+
+```bash
+$ kubectl delete mysqlopsrequest -n demo mysql-inplace-restore
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)